### PR TITLE
Currency selector

### PR DESCRIPTION
**Pull request by acm-sp85**
- Installed Ant dependencies
- Encapsulated the currency price as a separate function
- Added a dropdown with different currencies to play with. 
- Created a Credit system that starts with 3 lives, and decreases every time the player loses a bet

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -31,34 +34,44 @@ const StyledResultsContainer = styled.div`
...
     setBetPrice(formattedCurrency);
     if (referencePrice > formattedCurrency && betType === UP_TYPE) {
       setMessage('You loose!');
+      setCredits(credits - 1);
     } else if (referencePrice < formattedCurrency && betType === DOWN_TYPE) {
       setMessage('You loose!');
+      setCredits(credits - 1);
     } else if (referencePrice === formattedCurrency) {
       setMessage("It's a tie, try again");
     } else {
```
Comment (roninCode) [inline src/App.js:77]: You win! 
Maybe we should increment the credits they have? 

A fun feature we could implement is, say you get to 10 credits - then we show confetti or some animation. 
"Woohoo! You're the smartest person alive!"

File: src/App.js [hunk 2]
```diff
@@ -31,34 +34,44 @@ const StyledResultsContainer = styled.div`
...
-  // function to setReferencePrice when button onClick
+  // function to setReferencePrice when Button onClick
   const selectReference = () => {
     setReferencePrice(formattedCurrency);
     console.log('reference: ' + formattedCurrency);
   };
 
+  // function to select t(he currency
+  const selectCurrency = (e) => {
+    console.log(e.target.value);
```
Comment (roninCode) [inline src/App.js:60]: I'd remove the console logs from prs. 
A fun thing you can do (not for now) is setup different environments - i.e. staging vs. production. 

You can then write utils that will only console log if you are in staging and/or if you are an admin (but admin === auth and idk if we want to go down that road right now)

File: src/App.js [hunk 3]
```diff
@@ -1,8 +1,11 @@
-import BitcoinPrice from './components/BitcoinPrice';
+import CurrencyPrice from './components/CurrencyPrice';
 import './App.css';
 import { useState } from 'react';
 import Results from './components/Results';
-import styled, { css } from 'styled-components';
+// added Button from Antd library, at the moment is not really kicking in
+import { Button } from 'antd';
+// import styled, { css } from 'styled-components';
```
Comment (roninCode) [inline src/App.js:7]: I try to delete old code (not leave commented) to keep the files tight

**Review by roninCode — APPROVED** [review]
Nice! Great job getting antd to work! 

The comments I have are non-blocking - let's dive in more next week

File: src/App.js (hunk 1 above)
Comment (acm-sp85) [reply src/App.js:77]: yeah! let's do that :)

**Pull request merged**